baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/StrictUnusedVariable.java

```diff
@@ -253,28 +251,14 @@ public Description matchCompilationUnit(CompilationUnitTree tree, VisitorState s
                 // If we can not find the parameter in the owning method, then it must be a parameter to a lambda
                 // defined within the method
                 if (index == -1) {
-                    fixes = buildUnusedLambdaParameterFix(symbol, entry.getValue(), state);
+                    reportUnusedLambdaParameter(symbol, unused, specs, state);
                 } else {
-                    fixes = buildUnusedParameterFixes(symbol, methodSymbol, allUsageSites, state);
+                    reportUnusedParameter(symbol, methodSymbol, unused, allUsageSites, state);
                 }
             } else {
-                fixes = buildUnusedVarFixes(symbol, allUsageSites, state);
+                reportUnusedAssignment(symbol, unused, specs, allUsageSites, state);
```

Comment:
This only reports a match if the var is a local var, so no need to do it for all vars (including parameters) as we were before.